test/minitest/deprecation_toolkit_plugin_test.rb

```diff
@@ -5,7 +5,11 @@
 
 module Minitest
   class DeprecationToolkitPluginTest < ActiveSupport::TestCase
-    include ActiveSupport::Testing::Isolation
+    if Gem.loaded_specs["activesupport"].version.to_s >= "5.0"
```

Comment:
Can you use `ActiveSupport.gem_version` instead

Reply:
That's nice! Didn't know that was a thing.